ScanToShop/Managers/AlertManager.h

```diff
@@ -10,9 +10,16 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface AlertManager : UIViewController
+@interface AlertManager : NSObject
 
-+ (void)loginAlert:(NSString *)errorString ViewController:(UIViewController *) vc;
+typedef NS_ENUM(NSInteger, errorType) {
+    LoginErrorMissingInput,
+    ParseBackendError,
+    PasswordError,
+    SpaceNewlineError
```

Comment:
Maybe something like `InputValidationError` since you may have more types of input failure. Like, phone numbers needing to be digits or something.